packages/babel-helper-replace-supers/package.json

```diff
@@ -20,6 +20,12 @@
     "@babel/traverse": "workspace:^",
     "@babel/types": "workspace:^"
   },
+  "peerDependencies": {
+    "@babel/core": "^7.0.0-0"
```

Comment:
In theory adding a peer dependency is a breaking change, however this is an internal package only used by other packages that already depend on `@babel/core`, so it should be fine.

Could you remove the `@babel/traverse` and `@babel/types` dependencies, and import them from `@babel/core`? Or as an alternative, we could just add `@babel/template` as a dependency and avoid adding `@babel/core`.

Reply:
Now `template` is imported directly from `@babel/template`.

By the way, it seems that the original code (on `main` branch) of `@babel/helper-replace-supers` have an `import type { File } from "@babel/core";` without declaring `@babel/core` dependency in `package.json`. Does it matter?

https://github.com/babel/babel/blob/65c8496b8403aecb5e8bfc580c38321021d1ca16/packages/babel-helper-replace-supers/src/index.ts#L19

Reply:
It doesn't matter because we remove type-only imports before publishing, so the published code is fine.